test/vmtester/tests.cpp

```diff
@@ -104,12 +104,6 @@ TEST_F(evmc_vm_test, execute_create)
         read_buffer(result.output_data, result.output_size);
     }
 
-    if (result.status_code == EVMC_SUCCESS)
-    {
-        // This assumes that CREATE returns a non-zero address on success.
```

Comment:
Why remove this?

Reply:
This is wrong test. VM will not set the create address in the result. It didn't cause problems before because the `result.status_code == EVMC_SUCCESS` is never true for EVM not Ewasm.

Reply:
> VM will not set the create address in the result. 

~~What do you mean by this?~~

Ah right, we need to make this more explicit in the documentation.

Reply:
I will pull the commit to another PR with documentation updated.